src/components/Alert.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Alert } from 'react-bootstrap';
+
+const AlertComponent = (props) => {
+  const { variant, alertMsg } = props;
```

Comment:
You can destructure directly in the function parameters without creating a new variable.